packages/oasgraph/src/resolver_builder.ts

```diff
@@ -181,6 +185,44 @@ export function getResolver ({
       qs: query
     }
 
+    /**
+     * Deep merge optional "requestOptions" parameter with current request options.
+     * See https://github.com/TehShrike/deepmerge
+     */
+    if (!!requestOptions) {
```

Comment:
Is this double negation necessary?

Reply:
It's considered a best praticte by some (see http://airbnb.io/javascript/#coercion--booleans) to express that we only care about the boolean outcome.

But yes, in the case of an if statement it's not necessary ! It's just a habit of mine to use it :D I will change it.

Reply:
Oh interesting! I learned something new today. I will look into this in my own time. However, I believe we should remove it as this is the first time this pattern has been used in our code base. Hope you understand!